_docs/5_SubspaceNetwork/README.md

```diff
@@ -98,6 +109,17 @@ server {
   root /var/www/html;
   index index.html;
 
+  location /farmer-rpc {
+    proxy_buffering off;
+    proxy_pass http://127.0.0.1:9955/;
+    proxy_set_header X-Real-IP $remote_addr;
+    proxy_set_header Host $host;
+    proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+    proxy_http_version 1.1;
+    proxy_set_header Upgrade $http_upgrade;
+    proxy_set_header Connection "upgrade";
+  }
```

Comment:
Can we have `/farmer` or `/farmer-ws` and add `/ws` and `/http` for node RPC endpoints, so it is consistent.

Reply:
Sure, i updated this, to use /farmer instead and added /http enpoint.